assignments/array-methods.js

```diff
@@ -56,28 +56,37 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+let runnerName = runners.forEach((data) => fullName.push(`${data.first_name} ${data.last_name}`));
```

Comment:
Using generic variable names like `data` is great if you plan to refer back to your notes later on. However, it's good to get in the habit of using descriptive names like `person` or `runner` (or whatever makes sense to you). This will make your code more readable and easier to maintain.